src/normal.cc

```diff
@@ -383,6 +383,9 @@ void view_commands(Context& context, NormalParams params)
 
         const BufferCoord cursor = context.selections().main().cursor();
         Window& window = context.window();
+        const DisplayCoord max_offset{(window.dimensions().line - 1)/2, (window.dimensions().column - 1)/2};
+        const DisplayCoord scrolloff =
```

Comment:
I dont think this is correct, you want the min of lines and the min of columns, but here `min({1_line, 3_col}, {2_line, 1_col})` is `{1_line, 3_col}`, not {1_line, 1_col}`

Reply:
You are absolutely right. Perhaps i should modify the bindings `v<` and `v>` to account column offsets as well